aws_lambda_builders/utils.py

```diff
@@ -0,0 +1,54 @@
+"""
+Common utilities for the library
+"""
+
+import shutil
+import os
+
+
+def copytree(source, destination, ignore=None):
+    """
+    Similar to shutil.copytree except that it removes the limitation that the destination directory should
+    be present.
+
+    :type source: str
+    :param source:
+        Path to the source folder to copy
+
+    :type destination: str
+    :param destination:
+        Path to destination folder
+
+    :type ignore: function
+    :param ignore:
+        A function that returns a set of file names to ignore, given a list of available file names. Similar to the
+        ``ignore`` property of ``shutils.copytree`` method
+    """
+
+    if not os.path.exists(destination):
+        os.makedirs(destination)
+
+        try:
+            # Let's try to copy the directory metadata from source to destination
+            shutil.copystat(source, destination)
+        except WindowsError:  # pylint: disable=undefined-variable
+            # Can't copy file access times in Windows
+            pass
```

Comment:
Is this safe?  It's not clear to me what we are smothering with this pass statement.

Reply:
Not sure. I'll just add a debug log here instead